setupTests.js

```diff
@@ -51,7 +51,22 @@ Object.defineProperty(window, 'location', {
   },
   writable: true,
 });
-window.__PRELOADED_STATE__ = { random: 'Preloaded state, baby!' };
+
+function toArrayBuffer(buffer) {
```

Comment:
Explain this. You might also consider adding a docstring here. It's not immediately clear how and why it's used

Reply:
looks like it should go to utils?